File: hw/rojas/u1/HW11ChickenCoops/Poop.js
class Poop {
  constructor(id, chickenId) {
    this.id = id;
    this.chickenId = chickenId;
  }

  toString() {
    return `Poop#${this.id} from Chicken#${this.chickenId}`;
  }
}

export { Poop };

File: hw/aymacana/u1/hw14Chickenfarm/Java/src/model/Poop.java
package model;
/**
 *
 * @author Mateo Aymacaña
 */
public class Poop {
    private int amount;
    
    @Override
    public String toString(){
        return "Poop {" + "amount = " + amount + "}";
    }
    
    public Poop(int amount){
        this.amount = amount;
    }
    
    public int getAmount(){
        return amount;
    }
  
}
